_includes/cockroachcloud/cockroachcloud-pricing.md

```diff
@@ -1,10 +1,10 @@
 The choice of the cloud provider decides the price per node. For pricing comparison, refer to the following table:
 
- Hardware configuration            | AWS IOPS                        | GCP Pricing (per node, per hour)	  | AWS Pricing (per node, per hour) 
------------------------------------+---------------------------------+------------------------------------+------------------------------------
-`Option 1` (2 vCPU, 60 GiB disk)   | 600	                           | $0.50	                            | $0.55
-`Option 2` (4 vCPU, 150 GiB disk)  | 900	                           | $0.89                            	| $1.02
-`Option 3` (8 vCPU, 500 GiB disk)  | 1800	                           | $1.78                              | $2.00
-`Option 4` (16 vCPU, 900 GiB disk) | 2000	                           | $3.38                              | $3.83
+ Hardware configuration            | GCP Pricing (per node, per hour)	  | AWS Pricing (per node, per hour) 
+-----------------------------------+------------------------------------+------------------------------------
+`Option 1` (2 vCPU, 60 GiB disk)   | $0.50	                            | $0.55
```

Comment:
nit: the tab spacing is off for AWS pricing on this row.

Reply:
can't see the difference in atom for some reason, hopefully this fixes it